<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGroupBox" name="ZarobskiBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>90</y>
      <width>291</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Przychody</string>
    </property>
    <widget class="QPushButton" name="addIncomeButton">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>200</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Dodaj</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>90</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QDateEdit" name="dateEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QComboBox" name="incomeCategoryComboBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="WydatkiBox">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>90</y>
      <width>281</width>
      <height>231</height>
     </rect>
    </property>
    <property name="title">
     <string>Wydatki</string>
    </property>
    <widget class="QDateEdit" name="dateEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>130</y>
       <width>110</width>
       <height>25</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>90</y>
       <width>113</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="addExpenseButton">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>200</y>
       <width>80</width>
       <height>24</height>
      </rect>
     </property>
     <property name="text">
      <string>Dodaj</string>
     </property>
    </widget>
    <widget class="QComboBox" name="expenseCategoryComboBox">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>72</width>
       <height>24</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="stanKonta">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>370</y>
      <width>151</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="graph">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Graph</string>
    </property>
   </widget>
   <widget class="QPushButton" name="viewTableButton">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>360</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>Tabela</string>
    </property>
   </widget>
   <widget class="QPushButton" name="usun">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>151</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Usuń całą baze danych</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
